recipe/meta.yaml

```diff
@@ -249,31 +273,13 @@ outputs:
         - ninja         # [win]
         - llvm-openmp   # [linux and ((blas_impl == "openblas") or (blas_impl == "mkl"))]
       host:
-      # Building with blis fails due to a conda-build bug
-      {% if blas_impl != 'blis' %}
-        - {{ pin_subpackage("liblapack", exact=True) }}
-        - {{ pin_subpackage("liblapacke", exact=True) }}
-        - {{ pin_subpackage("libcblas", exact=True) }}
-        - {{ pin_subpackage("libblas", exact=True) }}
-        - openblas *={{ openblas_type }}*   # [win and blas_impl == 'openblas']
-      {% else %}
-        - blis 0.9.0
-      {% endif %}
+        - {{ pin_subpackage("blas-devel", exact=True) }}
+        - openblas =*={{ openblas_type }}*  # [win and blas_impl == 'openblas']
       run:
-        - {{ pin_subpackage("liblapack", exact=True) }}      # [blas_impl != 'blis']
-        - {{ pin_subpackage("liblapacke", exact=True) }}     # [blas_impl != 'blis']
-        - liblapack  {{ version }} *netlib                   # [blas_impl == 'blis']
-        - liblapacke {{ version }} *netlib                   # [blas_impl == 'blis']
-        - {{ pin_subpackage("libcblas", exact=True) }}
-        - {{ pin_subpackage("libblas", exact=True) }}
         - {{ pin_subpackage("blas-devel", exact=True) }}
     test:
       commands:
-        - test -f $PREFIX/lib/liblapacke.so                          # [linux]
-        - test -f $PREFIX/lib/liblapacke.so.{{ version_major }}      # [linux]
-        - test -f $PREFIX/lib/liblapacke.dylib                       # [osx]
-        - test -f $PREFIX/lib/liblapacke.{{ version_major }}.dylib   # [osx]
-        - if not exist %LIBRARY_BIN%/liblapacke.dll exit 1           # [win]
```

Comment:
Let's keep these

Reply:
Thanks for the review! I've kept these, but in `blas-devel` (which `blas`now wraps)

Reply:
That's fine, but it's better to have the tests here in addition to the ones in `blas-devel`.

Reply:
I'll reinstate, but I don't understand... If the files are actually contained in `blas-devel`, and `blas` is a pure wrapper-package with no content, tests on the wrapper are redundant with the tests for the package being wrapped.